gulpfile.coffee

```diff
@@ -45,15 +46,19 @@ gulp.task 'test', ->
 		}))
 
 gulp.task 'build', (callback) ->
-	runSequence('build-node', 'build-browser', callback)
+	runSequence('build-node', 'pack-browser', callback)
 
+gulp.task 'inject-version', ->
+	gulp.src(["#{OPTIONS.directories.build}util/sdk-version.js"])
+		.pipe(replace("var sdkVersion = '0.0.0';", "var sdkVersion = '#{packageJSON.version}';"))
```

Comment:
// TODO: use a better build step for the node sources, that supports this kind of stuff.
Eg: rollup

Reply:
You could use some magic value for the version string like `__REPLACE_WITH_SDK_VERSION__` and replace this string instead of the whole line, since I guess it depends on the ts to js compilation?
```
.pipe(replace("__REPLACE_WITH_SDK_VERSION__", "#{packageJSON.version}"))
```